pkg/storage/client_raft_test.go

```diff
@@ -1126,6 +1127,144 @@ func TestRefreshPendingCommands(t *testing.T) {
 	}
 }
 
+// Test that when a Raft group is not able to establish a quorum, its Raft log
+// does not grow without bound. It tests two different scenerios where this used
```

Comment:
scenerios 💩

Reply:
Way ahead of you https://github.com/cockroachdb/cockroach/commit/d41111da6740b3e07bdf9f00724606b455e6e11f#diff-b1bd51088f60beeacafe950f5bc700f8R1131 😄